src/main/java/org/folio/marccat/resources/TenantAPI.java

```diff
@@ -25,29 +34,30 @@
   private TenantRefService tenantRefService;
 
   @PostMapping
-  public ResponseEntity<String> create(
-    @RequestHeader(Global.OKAPI_TENANT_HEADER_NAME) String tenant,
-    @RequestHeader(Global.OKAPI_URL) String okapiUrl,
-    @RequestHeader(Global.OKAPI_TO_URL) String okapiUrlTo,
-    @RequestBody TenantAttributes attributes
-  ) throws SQLException, IOException {
+  public ResponseEntity<String> create(@RequestHeader(Global.OKAPI_TENANT_HEADER_NAME) String tenant,
+      @RequestHeader(Global.OKAPI_URL) String okapiUrl, @RequestHeader(Global.OKAPI_URL_TO) String okapiUrlTo,
+      @RequestBody TenantAttributes attributes) throws SQLException, IOException {
```

Comment:
Remove the two exceptions (SQLException, IOException ) and handle them with try, catch. Handle them with an error like HttpStatus.INTERNAL_SERVER_ERROR.  An example class BibliographicRecordAPI the save () method.

Reply:
Done